common/src/chain/config.rs

```diff
@@ -61,7 +61,7 @@ impl ChainConfig {
     }
 }
 
-const MAINNET_ADDRESS_PREFIX: &str = "mlt";
+pub const MAINNET_ADDRESS_PREFIX: &str = "ml";
```

Comment:
Is this something we agreed about before? During my vacation I saw Lukas mention it and couldn't get into it. Actually I think both mlt and ml are bad because the letter `l` is one of the bad letters, that can be interpreted as `l`, `i` and `1` depending on how you write it, hence dropping these characters from base58 and bech32. I would prefer to go for `mt` to be honest. @muursh and @iljakuklic can maybe support this idea.

Reply:
Also, btw, this is NOT pub. You're supposed to take this from ChainConfig.

Reply:
I'll support something without `l` for sure. `mt` works for me.

Reply:
I was going by @erubboli's Twitter banner, the _Only Source of Truth_ ™, I believe `mt` was originally for testnet so that would have to be changed too (`mtt` maybe?)

Reply:
`ml` was the original plan but I agree we should avoid dodgy letters. Agreed that it also means the testnet hrp needs changing. 

What about `mnt` for `mint` on the mainnet and `tm` on the testnet?

Reply:
I'm fine with `mn` to be fair

Reply:
Done